src/js/components/progress-bar.js

```diff
@@ -0,0 +1,21 @@
+import { createComponent } from '../mini';
+import { connect } from '../store';
+import { totalRebusesCount } from '../rebuses';
+
+export function ProgressBar() {
+  return connect(
+    createComponent({
+      render() {
```

Comment:
You can use the props that will be passed to the render method instead,
```js
render({ rebuses }) {....
```

and then figure out how many rebuses that has been answered by doing,
```js
const answeredRebuses = rebuses.filter(rebus => rebus.isAnswered);
```

this way we don't have to get anything from localStorage, but can instead rely on the appStores state